src/StereoDB/Infra/Sql/Sql.fs

```diff
@@ -99,6 +99,13 @@ module internal QueryBuilder =
                         if columnProperty <> null then
```

Comment:
try to use expressions that use simpler to read '=' than '<>'
also, consider using standard F# function **isNull** or **if not (isNull value)**
null - is evil even as text token :)